raaghu-elements/src/rds-stepper/rds-stepper.tsx

```
@@ -208,25 +212,25 @@ const RdsStepper = (props: RdsStepperProps) => {
                         {[...Array(props.steps)].map((_, index) => (
                             <div
                                 key={index}
-                                className={`block_step d-flex align-items-center justify-content-center ${props.state === 'default'
+                                className={`block_step d-flex align-items-center justify-content-center ${props.stepDetailsName && props.stepDetailsName[index].stepState === 'default'
```

ESLint

[error] Strings must use doublequote.

(quotes)